lib/galaxy/web/base/controllers/admin.py

```diff
@@ -1123,6 +1123,24 @@ def job_info( self, trans, jobid=None ):
                                     job=job,
                                     message="<a href='jobs'>Back</a>" )
 
+    @web.expose
+    @web.require_admin
+    def sanitize_whitelist( self, trans, submit_whitelist=False, tools_to_whitelist=[]):
+        if submit_whitelist:
+            with open(trans.app.config.sanitize_whitelist_file, 'wt') as f:
+                if isinstance(tools_to_whitelist, basestring):
+                    tools_to_whitelist = [tools_to_whitelist]
+                # write config/sanitize_whitelist.txt file with new whitelist and update in-memory list.
```

Comment:
Ahh, good point, forgot to update my commentary when I swapped this to use the configfile (which by default is config/sanitize_whitelist.txt).  Updating.